### ISSUE TITLE
Fix efiling filter bugs

### ISSUE THREAD

**Issue by noahmanger** [18F/openFEC#2157]
As I'm QA'ing the raw efiling data, I'm seeing a few filter bugs that need to be resolved for release (either by fixing them or removing the filters from the front-end):

**[Receipts](https://fec-dev-proxy.18f.gov/data/receipts/)**
- [x] Contributor employer
- [x] Contributor occupation

**[Disbursements](https://fec-dev-proxy.18f.gov/data/disbursements/)**
- [x] Recipient name

**[IEs](https://fec-dev-proxy.18f.gov/data/independent-expenditures/)**
- [x] Candidate mentioned
- [x] Expenditure date

**Comment by jontours** [comment]
I'll comment on these as I take care of them, but I'm pretty sure `candidate_name` filter is working, do you have an example?  Expenditure date is resolved by PR just opened against the release branch.

**Comment by jontours** [comment]
Oh, I'd say the fix for the other filters is to remove them temporarily, as I documented here https://github.com/18F/openFEC-web-app/pull/1771: 
```
I took a look at the new filters and verified most work, but wanted to make note of something, or solicit feedback on these filters: contributor_employer and contributor_occupation. For the processed these are tsvectors which are essentially vectorized in the database to enable quick search in the database. It's easy to do that for the processed because we generate the tables and the columns therein.

For the raw resources it's not quite as simple. However, to start cautiously I was able to add a column to raw resource that updates as rows are updated or inserted in the efiling table. Since it's a big table and currently not sure how performance will be affected on dev I added a trigger and column for contributor_name. If we don't see any negative performance or noted degradation on production with this process then I think we can go ahead and add columns and triggers for occupation and employer.
```

The same goes for recipient_name for disbursements, I think it may be best to see if there are no performance issues on production than go ahead and add the rest of those filters.  But if you want we can go ahead and add the triggers now.

**Comment by jontours** [comment]
The pertinent paragraph for the yet to be implemented filters is this: 

I took a look at the new filters and verified most work, but wanted to make note of something, or solicit feedback on these filters: contributor_employer and contributor_occupation. For the processed these are tsvectors which are essentially vectorized in the database to enable quick search in the database. It's easy to do that for the processed because we generate the tables and the columns therein.

For the raw resources it's not quite as simple. However, to start cautiously I was able to add a column to raw resource that updates as rows are updated or inserted in the efiling table. Since it's a big table and currently not sure how performance will be affected on dev I added a trigger and column for contributor_name. If we don't see any negative performance or noted degradation on production with this process then I think we can go ahead and add columns and triggers for occupation and employer.

**Comment by noahmanger** [comment]
Just to close the loop on this, should I just remove the filters from the front-end?

**Comment by jontours** [comment]
I think it may be best.  If we see that the new indices on production don't cause any issues then we can add the additional filters back in.  I'm hopeful they don't cause problems.

**Comment by noahmanger** [comment]
Resolved by #1800
